src/SelfServe/SelfServeTypes.ts

```diff
@@ -126,5 +126,5 @@ export interface SelfServeNotification {
 
 export interface RefreshResult {
   isUpdateInProgress: boolean;
-  notificationMessage: string;
+  notificationMessage?: string;
```

Comment:
change this back. This is a required message to show a notification when the refresh is taking place. you can leave it as undefined when isUpdatingInProgress is false.

Reply:
Okay, this is not Sqlx specific I see. That makes sense